tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_prefetcher.cpp

```diff
@@ -461,15 +461,15 @@ void gen_dram_packed_read_cmd(Device *device,
     vector<CQPrefetchRelayPagedPackedSubCmd> sub_cmds;
 
     uint32_t total_length = 0;
-    for (auto l : lengths) {
-        total_length += l;
+    for (const auto& length : lengths) {
+        total_length += length;
     }
     gen_bare_dispatcher_unicast_write_cmd(device, dispatch_cmds, worker_core, device_data, total_length);
     add_prefetcher_cmd(prefetch_cmds, cmd_sizes, CQ_PREFETCH_CMD_RELAY_INLINE_NOFLUSH, dispatch_cmds);
 
     uint32_t page_size = 1 << log_page_size;
     int count = 0;
-    for (auto length : lengths) {
+    for (const auto& length : lengths) {
```

Comment:
why use a const ref when we just want a value (copying an int has no cost here)

Reply:
Ack